common/src/main/java/com/graphicsfuzz/common/util/PipelineInfo.java

```diff
@@ -226,14 +226,23 @@ private static String get(BasicType type, boolean isArray) {
     return result;
   }
 
-  public void addUniformBinding(String uniformName, int number) {
+  public void addUniformBinding(String uniformName, boolean pushConstant, int number) {
```

Comment:
Add JavaDoc explaining that `number` is the binding number, but that it is ignored if `pushConstant` is true.

Reply:
done